gdb/printing.py

```diff
@@ -261,20 +261,26 @@ def _get_children(self, max_count: int) -> None:
                     (c.GetName(), gdb.Value(c))
                     for c in self._sbvalue.GetNonSyntheticValue()]
             # Prepend errors (as synthetic children) to the list of children.
+            byte_order = self._sbvalue.GetTarget().GetByteOrder()
+            address_byte_size = self._sbvalue.GetTarget().GetAddressByteSize()
             for error_str in self._captured_errors:
+                error_str = 'GALA_ERROR(%s)GALA_ERROR' % error_str
```

Comment:
Please remove the `GALA_ERROR(...)GALA_ERROR` delimiters. These were meant for the C++ parser as a raw string delimiter that was very unlikely to collide with actual error strings, and were never seen by users. Now that we're not calling `CreateValueFromExpression` they shouldn't be here.

Actually, `error_str` should already be a string so now that we don't need the delimiters you can just remove the whole line.